app/src/main/java/com/example/news_articles/articlesApii/NewsService.java

```diff
@@ -0,0 +1,15 @@
+package com.example.news_articles.articlesApii;
+
+import com.example.news_articles.articles.ArticleNetworkResponse;
+
+import io.reactivex.rxjava3.core.Single;
+import retrofit.http.GET;
+import retrofit.http.Header;
+import retrofit.http.Query;
+
+public interface NewsService {
+
+    @GET("top-headlines?country=us")
```

Comment:
Move the country to a Query param, when the service will be used thats when you will have to pass the default US.